cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -177,20 +177,82 @@ jobs:
           path: ${{ steps.cargo-dist.outputs.paths }}
 {{%- endif %}}
 
+{{%- if sslcom_windows_sign %}}
+
+  # Sign Windows artifacts with ssl.com
+  sign-windows-artifacts:
+    needs:
+      - plan
+      - upload-local-artifacts
+    {{%- if global_task %}}
+      - upload-global-artifacts
+    {{%- endif %}}
+    runs-on: "ubuntu-20.04"
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      SIGN_DIR_IN: target/distrib/sign-input
+      SIGN_DIR_OUT: target/distrib/sign-output
+    steps:
+      # Get all the artifacts for the signing tasks to use
+      - name: Fetch local artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: artifacts
+          path: target/distrib/
+      # Only try to sign files that the tool can handle
+      - name: Select Signable Artifacts
+        run: |
+          mkdir -p "$SIGN_DIR_IN"
+          mkdir -p "$SIGN_DIR_OUT"
+          for file in target/distrib/*.{msi,ps1}; do
+            [[ -e $file ]] && mv "$file" "$SIGN_DIR_IN" && echo "signing $file";
+          done
+      # Sign the files
+      - name: Sign Artifacts with CodeSignTool
+        uses: sslcom/esigner-codesign@develop
+        with:
+          command: batch_sign
+          username: ${{ secrets.SSL_CODESIGN_USER }}
```

Comment:
I think using an `SSLCOM` prefix would make this a little clearer to me.

Reply:
I changed everything to SSLDOTCOM/ssldotcom prefixes (matches how we treat axo.dev) and made these match a bit more with the action's defaults.